apparmor.d/groups/apt/apt-cache

```diff
@@ -12,10 +12,15 @@ profile apt-cache @{exec_path} {
   include <abstractions/base>
   include <abstractions/apt>
   include <abstractions/consoles>
+  include <abstractions/evince>
 
   @{exec_path} mr,
 
   @{bin}/dpkg rPx -> child-dpkg,
+  @{bin}/gnufalse mrix,
+  @{bin}/gnutrue mrix,
+
+  @{tmp}/tmp.*/rootdir/usr/bin/dpkg mrix,
```

Comment:
Executable `/tmp`... Any user could write into this path.

Reply:
`tmp.*/rootdir/usr/bin/dpkg mrix,` I'm inclined to believe this is something apt does for testing and not related to how this application functions outside of that. 

https://codesearch.debian.net/search?q=getaptconfig+package%3A%5CQapt%5CE&literal=1
https://codesearch.debian.net/search?q=APT_CONFIG+package%3A%5CQapt%5CE&literal=1